inspect/README.md

```diff
@@ -0,0 +1,195 @@
+# forta-inspector-bot
+
+Inspects run environment capabilities and limits
+
+Moved from [forta-network/forta-inspector-bot](forta-inspector-bot-repo)
+
+# Inspection Details
+
+## Scan API Inspection
+
+### `scan-api.accessible`
+
+Checks Scan APIs accessibility by establishing rpc connection and asking for current block height
+
+`1` for accessible, `-1` for not
+
+### `scan-api.module.eth`
+
+The scan node should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.module.net`
+
+The scan api should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.block-by-number.hash`
+
+The hash of the block data retrieved from the scan API
+
+## Trace API Inspection
+
+### `trace-api.accessible`
+
+Checks trace APIs accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `trace-api.supported`
+
+Only necessary for Ethereum Mainnet.
+
+Returns `1` by default on other chains
```

Comment:
There is a `CheckTrace` field in inspection config. Depending on this, the inspection is able to skip checking trace functionality. But the feature is marked as failing/not available in that case. We can remove this line and keep the consumer-side expectation simple.